libraries/azure-app-configuration-importer/src/errors.ts

```diff
@@ -39,3 +41,31 @@ export class OperationTimeoutError extends Error {
  * Custom error type used when null arguments are passed
  */
 export class ArgumentNullError extends Error {}
+
+export interface AjvError {
```

Comment:
This is not an `Error`, this is just a wrapper of error information right? How about call it `AjvErrorInfo` to differentiate from the real Error?

Reply:
Yes its just a wrapper. Updated